recipes/sdt-python/meta.yaml

```diff
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: 2020 Lukas Schrangl <lukas.schrangl@tuwien.ac.at>
+#
+# SPDX-License-Identifier: BSD-3-Clause
+{% set name = "sdt-python" %}
+{% set version = "14.3.2" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/schuetzgroup/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: c331494130284d6360e68de21a5ce0092988b2e16f34594dc7445a3fc1029c13
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+    noarch: python
+
+requirements:
+    host:
+        - python
+        - pip
+    run:
+        - python
+        - numpy >=1.10
+        - pandas
+        - pytables
+        - scipy >0.18
+        - tifffile >=0.14.0
+        - pyyaml
+        - pims >=0.3.0
+        - numba
+        - qtpy >=1.1
+        - matplotlib
+        - pywavelets >=0.3.0
+
+test:
+    requires:
+        - pytest
+        - trackpy
+        - lmfit
+    commands:
+        - pytest -v tests
+    source_files:
+        - tests
+    imports:
+        - sdt
+        - sdt.brightness
+        - sdt.changepoint
+        - sdt.chromatic
+        - sdt.exp_fit
+        - sdt.flatfield
+        - sdt.fret
+        - sdt.gaussian_fit
+        - sdt.helper
+        - sdt.image
+        - sdt.io
+        - sdt.loc
+        - sdt.motion
+        - sdt.multicolor
+        - sdt.roi
+        - sdt.sim
+        - sdt.spatial
+
+about:
+    home: https://github.com/schuetzgroup/sdt-python
+    license: BSD-3-Clause
```

Comment:
```suggestion
    license: BSD-3-Clause
    license_file: THE_NAME_OF_THE_FILE_IN_THE_SDIST
```

Reply:
@crisburr So license files should be included via `license_file` even though they already included by the package itself via the `metadata/license_files` key in setup.cfg?

Reply:
Yes

Reply:
Done.